src/msi_solver.cc

```diff
@@ -26,7 +26,7 @@ using std::complex;
 #endif
 
 // ***********************************
-// 		MSI_SOLVER
+//    MSI_SOLVER
```

Comment:
This file appears to only have whitespace changes. If formatting needs to be fixed (tabs to two spaces, etc.) then please submit a separate PR with commits that specifically address it.